lib/src/view/play/challenge_screen.dart

```diff
@@ -406,15 +407,47 @@ class _ChallengeBodyState extends ConsumerState<_ChallengeBody> {
                                   );
                                 }
                               : null
-                          : snapshot.connectionState == ConnectionState.waiting
-                              ? null
-                              // TODO handle correspondence time control
-                              : () async {
+                          : timeControl ==
+                                  ChallengeTimeControlType.correspondence
+                              ? () async {
+                                  final createGameService =
+                                      ref.read(createGameServiceProvider);
                                   showPlatformSnackbar(
                                     context,
-                                    'Correspondence time control is not supported yet',
+                                    'Sent challenge to ${widget.user.name}',
```

Comment:
Have you implemented sending correspondence challenges?

I would prefer to keep one PR per feature, to simplify the review process. This one is about receiving challenges, not sending them.

Could you open another one later with the challenge creation, and also send a screen recording to show how it works? thanks.

Reply:
Yeah i think when i was rebasing to remove the localisation changes i accidentally removed the commit that modified the user screen to actually implement this. the implementation is already there in the comment from before left by someone else it just needs to be uncommented out. 
If you feel it needs its own PR im happy to open another one.

Reply:
i just uncommented it out as its not a big change